package/yast2-packager.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Oct  3 16:38:36 UTC 2016 - jreidinger@suse.com
+
+- prevent double URL encoding for ISO (bsc#954813)
+- 3.1.118
```

Comment:
I'd revert the version bump, I'd do that just before releasing the maintenance update (to avoid possible collisions later).